crypto/keys/internal/ecdsa/privkey_internal_test.go

```diff
@@ -63,4 +64,29 @@ func (suite *SKSuite) TestSign() {
 	// Mutate the message
 	msg[1] ^= byte(2)
 	require.False(suite.pk.VerifySignature(msg, sig))
+
```

Comment:
I believe you need to

```suggestion
	msg[1] ^= byte(2) // reset to original message
```

or else VerifySignature==false not because of normalization check

Reply:
I moved the low-s sig check before the msg mutation check. Thank you!!!